src/develop/lightroom.c

```diff
@@ -810,12 +811,57 @@ static void _lrop(const dt_develop_t *dev, const xmlDocPtr doc, const dt_imgid_t
         data->has_rating = TRUE;
       }
     }
+    else if(!xmlStrcmp(name, (const xmlChar *)"GPSLatitudeRef"))
+    {
+      if g_str_equal("N", (const char *)value)
```

Comment:
This is not C and won't compile. So I bet it has not been tested.

Reply:
Let me take a look, I did in fact build and use it.  So let me see what the issue is.